app/models/user.rb

```diff
@@ -1,7 +1,10 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
Great job team 🤝
- As stated in the project requirements, kindly ensure the user is able to log in and register into the application successfully. This is not possible at the moment. Kindly fix this issue and resubmit your work.
- As stated in the project requirements, kindly ensure your login and registration pages replicate the wireframes provided. Kindly take a look at this [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) for more clarity.